<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_pause">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_easy">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>一般</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_midle">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>中等</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_hard">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>很难</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>分数:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_score">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_change_clr">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>切换颜色</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
